opengrok-web/src/main/java/org/opengrok/web/servlet/MetricsServlet.java

```diff
@@ -39,7 +39,7 @@
     @Override
     protected void doGet(final HttpServletRequest req, final HttpServletResponse resp) throws IOException {
         try (PrintWriter pw = resp.getWriter()) {
-            pw.print(Metrics.getPrometheusRegistry().scrape());
+            pw.print(Metrics.getInstance().getPrometheusRegistry().scrape());
```

Comment:
So unless `PROMETHEUS` then `NullPointerException` here?

Reply:
This was handled before however got lost with rebasing/refactoring, reverting.